docs-2.0/synchronization-and-migration/replication-between-clusters.md

```diff
@@ -386,9 +386,11 @@ The schema data is synchronized through the Meta listener, while the vertex/edge
 
 ### How to deal with synchronization failures?
```

Comment:
Can we change this title to like【What should we do once synchronization problems/issues are fixed?】
The current title lets me think of solutions to the problem instead of what we should do after the problem is fixed.

Reply:
Thx but that's not the question.